renderer/shared/providers/invite-context.js

```diff
@@ -76,8 +80,8 @@ function InviteProvider({children}) {
         const isNewInviteActivated = !!invitee
 
         const canKill =
-          !!invitee &&
           !!invitedIdentity &&
+          (!!invitee || invitedIdentity.state === IdentityStatus.Invite) &&

```

Comment:
not sure we need this

isn't it checked by the line below already? i mean this `invitedIdentity.state === IdentityStatus.Invite`